views/jira-configuration.hbs

```diff
@@ -321,8 +321,9 @@
                 Shortcut
               </span>
             </a>
+            <input type="hidden" id="_csrf" name="_csrf" value="{{../csrfToken}}">
             <button
-              class="ak-button ak-button__appearance-primary add-organization-link"
+              class="ak-button ak-button__appearance-primary {{handleNavigationClassName}}"
```

Comment:
Only modified the `Connect GH organization` button for empty state. The other button will be updated when we update the GH config screen to render multiple servers + cloud (button becomes a dropdown).